src/components/ecology.jsx

```diff
@@ -4,24 +4,28 @@ import Overview from "./overview";
 
 export default class Ecology extends React.Component {
   renderAPI(source) {
-    if (source) {
-      return (
-        <div className="Documentation">
-          <API source={this.props.source}/>
-        </div>
-      );
+    if (!source) {
+      return null;
     }
+
+    return (
+      <div className="Documentation">
+        <API source={source}/>
+      </div>
+    );
   }
   render() {
     return (
       <div className="Ecology">
         <div className="Overview">
           <Overview
+            copyToClipboard={this.props.copyToClipboard}
+            customRenderers={this.props.customRenderers}
             exportGist={this.props.exportGist}
             markdown={this.props.overview}
+            playgroundtheme={this.props.playgroundtheme}
```

Comment:
I'd destructure all the `this.props` things on top of `render` for tidyness *shrug*